tools/lsp/build.rs

```diff
@@ -2,5 +2,9 @@
 // SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
 
 fn main() {
-    println!("cargo:rustc-env=SLINT_ENABLE_EXPERIMENTAL_FEATURES=1");
+    // Make the compiler handle ComponentContainer:
+    // TODO: Remove if ComponentContainer anc component-factory no longer need this!
```

Comment:
```
```

I was about to offer a suggestion with the typo corrected. But I think this doesn't deserve a TODO. SLINT_ENABLE_EXPERIMENTAL_FEATURES is ok to define for our own things, and we'll grep for it if we remove it. The TODO don't add value, the previous line already explain why it's there.